library/std/src/path.rs

```diff
@@ -2405,6 +2419,23 @@ impl Path {
     ///
     /// This is a convenience function that coerces errors to false. If you want to
     /// check errors, call [`fs::metadata`].
+    ///
+    /// An example of proper error handling.
+    /// ```no_run
+    /// use std::path::Path;
+    /// use std::io::{ self, ErrorKind};
+    /// use std::fs;
```

Comment:
As formatted by rustfmt:

```suggestion
    /// use std::fs;
    /// use std::io::{self, ErrorKind};
    /// use std::path::Path;
```